lib/builders/texify.js

```diff
@@ -31,10 +31,12 @@ export default class TexifyBuilder extends Builder {
   }
 
   constructArgs (filePath) {
+    const enableSynctex = atom.config.get('latex.enableSynctex') !== false
+
     const args = [
       '--batch',
       '--pdf',
-      '--tex-option="--synctex=1"',
+      `--tex-option="--synctex=${enableSynctex ? 1 : 0}"`,
```

Comment:
See [the comment](https://github.com/thomasjo/atom-latex/pull/183/files#r65812979) for the equivalent change in latexmk.js.